testing/preview_all_cameras/preview_all_cameras.py

```diff
@@ -0,0 +1,27 @@
+#Example for visualization of all cameras from one device
+import cv2
+import depthai
+from depthai import node, message
+
+class all_cams(depthai.Node):
```

Comment:
The python way of styling the class name is
`AllCams`

the snake case you use here is used for function names